sections/selector.tex

```diff
@@ -4,13 +4,114 @@ \section{Writing a Selector}\label{sec:selector}
 The corresponding task is called \CCSPStlye{cf.SelectEvents}.
+
 For more information, please consider Ref.~\cite{cf_repo}.
 
-\begin{table}[t]
-	\Caption{Selection criteria for leptons}{Shown are the selection criteria for electrons (muons) at the 'loose' and 'tight'}
+\renewcommand{\arraystretch}{1.5}
+\begin{table}[h!]
+    \centering
+    \begin{tabular}{|m{4cm}|m{5cm}|m{5.5cm}|}
+    \hline
+    & \textbf{Electrons} & \textbf{Muons} \\ \hline
+    \textbf{Kinematic cuts} &
+    \begin{itemize}[leftmargin=*]
+    \item $p_T^e > 7$ GeV
+    \item $|\eta^e| < 2.5$
+    \end{itemize} &
+    \begin{itemize}[leftmargin=*]
+        \item $p_T^\mu > 5$ GeV
+        \item $|\eta^\mu| < 2.5$
+    \end{itemize} \\ \hline
+    \textbf{Vertex cuts} &
+    \begin{itemize}[leftmargin=*]
+        \item $d_{xy} < 0.5$
+        \item $d_z < 1$ cm
+        \item $SIP < 4$
+    \end{itemize} &
+    \begin{itemize}[leftmargin=*]
+        \item $d_{xy} < 0.5$
+        \item $d_z < 1$ cm
+        \item $SIP < 4$
+    \end{itemize} \\ \hline
+    \textbf{Isolation \& ID for \newline 'tight' working point} & Dedicated BDT targeting \newline prompt electrons. & Select only muons within \newline a well-defined cone ($R=0.35$). \\ \hline
+    \end{tabular}
+    \Caption{Selection criteria for leptons.}{Shown are the selection cuts for electrons/muons at the 'loose' working point, with the last row defining the extra requirement for the leptons to pass the 'tight' working point.}
+    \label{leptonSelection}
 \end{table}
+
 In this part of the tutorial, we will write selections for electrons and muons.
-\textbf{\underline{Loose Electrons}}
+In the script \code{h4l/selection/lepton.py} you can find the base structure to implement two \CCSPStlye{Selector} modules, \code{electron\_selection} and \code{muon\_selection}.
+Each of these objects uses the relevant event information for its implementation.
+
+
+\textbf{\underline{Electron Selection}}
+
+For \code{electron\_selection}, the electron kinematic information is first loaded into the \code{uses} set.
+Then, information that is dependent on the nanoAOD version is loaded. In this case, which MVA (Multi-Variate Analysis) flag should be used.
+Notice that we use the union operator \code{|} to append either \code{Electron.mvaFall17V2Iso} or \code{Electron.mvaHZZIso} to the set containing the kinematic variables.
+Lastly, to perform four-vector calculations, we also recquire \code{attach\_coffea\_behavior}, which is imported at the beginning of the script from \code{columnflow.selection.util}.
+
+
+This \CCSPStlye{Selector} object also has two more dependencies, \code{exposed} and \code{sandbox}.
+The first one determines whether or not the \CCSPStlye{Selector} object is available from the command line.
+As mentioned in the last section, \code{sandbox} specifies the software enviorment where this \CCSPStlye{Selector} will be executed.
```

Comment:
While this is a short and reasonable summary of these properties, it might also make sense to simply reference to the docs - we actually have a section on these keywords in the description of Selections